### ISSUE TITLE
Step forward is different with alt+l than with the button

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/molecules-and-light#360]
**Test device**
Dell
**Operating System**
Win 10
**Browser**
Chrome
**Problem description**
For https://github.com/phetsims/QA/issues/513.
Stepping forward with alt+l steps forward more than pressing the step forward button (with mouse or keyboard nav) does.

**Visuals**
Alt+l
![choppybyaltl](https://user-images.githubusercontent.com/41024075/87327210-051a6600-c4f1-11ea-9b3d-2e7af88693d6.gif)
Button
![smoothbyclicking](https://user-images.githubusercontent.com/41024075/87327228-0cda0a80-c4f1-11ea-8c00-85128ccda01b.gif)

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
Name: ‪Molecules and Light‬
URL: https://phet-dev.colorado.edu/html/molecules-and-light/1.5.0-rc.2/phet/molecules-and-light_all_phet.html
Version: 1.5.0-rc.2 2020-07-09 20:53:24 UTC
Features missing: touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.116 Safari/537.36
Language: en-US
Window: 1536x722
Pixel Ratio: 2.5/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by jessegreenberg** [comment]
Thanks for pointing out @KatieWoe but this behavior was actually requested in https://github.com/phetsims/molecules-and-light/issues/336#issuecomment-624242818. The reason is that with larger time steps it will be easier for screen reader users to receive feedback from molecule/photon interactions since there are less steps where nothing happens.

Closing this issue, but feel free to reopen if I misunderstood.
